OptimizelySDK/OptimizelyUserContext.cs

```diff
@@ -19,34 +19,51 @@
 using OptimizelySDK.ErrorHandler;
 using OptimizelySDK.Entity;
 using OptimizelySDK.OptimizelyDecisions;
+using System;
 
 namespace OptimizelySDK
 {
     /// <summary>
     /// OptimizelyUserContext defines user contexts that the SDK will use to make decisions for
     /// </summary>
-    public class OptimizelyUserContext
+    public partial class OptimizelyUserContext
```

Comment:
Why changed it to partial?

Reply:
looks like my editor config is adding things it shouldn't be!